dockstore-common/generated/src/main/resources/pom.xml

```diff
@@ -19,7 +19,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>io.dockstore</groupId>
   <artifactId>dockstore-common</artifactId>
-  <version>1.16.0-beta.6</version>
+  <version>1.16.0-SNAPSHOT</version>
```

Comment:
Hmm, not sure this should be changed. Maybe it should though, I'm a little unclear.

Reply:
I think it should be. In the [wiki](https://wiki.oicr.on.ca/display/DOC/Dockstore+Releases#DockstoreReleases-Maven-basedRepositories), it says to reset the generated files after pushing the new tag:

>reset generated files if the branch you're working on is going to be merged (like in hubflow release finish)
./mvnw clean install -DskipTests

The [build](https://circleci.com/gh/dockstore/dockstore/47742) also fails if I don't update the generated files

Reply:
Yeah, to confirm, I think tagged builds will get a "real" version. 
All other generated pom files will get a snapshot version. 
That said, for this change to show up in a "normal" PR, I think that means a step was skipped somewhere